### PR TITLE
Modal working with comparison and Removal of Your Outfit functionality now working.

### PR DESCRIPTION
**Pull request by Acid-Override**

### CODE REVIEW

File: client/src/components/RelatedItems/YourOutfit.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { useContext } from 'react';
+import _ from 'underscore';
```
Comment (addisonhernandez) [inline client/src/components/RelatedItems/YourOutfit.jsx:2]: it looks like underscore is not used in this file. Is it included here for future-proofing?

**Comment by addisonhernandez** [comment]
[Investigate set operations for RelatedItems/ComparisonModal](https://trello.com/c/RfQFyUzy/39-investigate-set-operations-for-relateditems-comparisonmodal)

**Review by addisonhernandez — APPROVED** [review]

**Comment by positivefx** [comment]
[Thurs, Apr 7- Daily Merge Party 🎉](https://trello.com/c/Ns3fSkSP/38-thurs-apr-7-daily-merge-party-%F0%9F%8E%89)

**Comment by Acid-Override** [comment]
[Comparison Modal](https://trello.com/c/gu0r1YDP/13-comparison-modal)

**Pull request merged**